spec/active_record/typed_store_spec.rb

```diff
@@ -14,6 +14,14 @@
 
   end
 
+  describe 'Marshal.dump' do
+
+    it 'dumps the model' do
+      Marshal.dump(model)
+    end
```

Comment:
sure.. Let me trace down the route cause.. it is possibly due to the new attributes API